moduleframework/mtf_scheduler.py

```diff
@@ -209,16 +217,27 @@ def __init__(self, args, unknown):
                 STATIC_LINTERS=common.conf["generic"]["static_tests"]))
         self.args = args
 
-        for param in unknown:
-            # take care of this, see tags for safe/unsafe:
-            # http://avocado-framework.readthedocs.io/en/52.0/WritingTests.html#categorizing-tests
-            testlist = glob.glob(param)
-            if testlist:
-                # this is list of tests in local file
+        # parse unknow options and try to find what parameter is test
+        while unknown:
+            if unknown[0] in self.A_KNOWN_PARAMS_SIMPLE:
```

Comment:
What is `unknown [0]`. I miss a documentation which will explain it.

Reply:
This is '-' in self.A_KNOWN_PARAMS_SIMPLE which is always true.

Reply:
I didn't changed this, ``unknow`` variable were in previous version. I've just little bit chaged logic how to split arguments of avocado and what are tests.